js/main.js

```diff
@@ -209,6 +223,65 @@ function editEntriesHandler(event) {
   const $h2FormTitle = document.querySelector('h2.form-title');
   $h2FormTitle.innerText = 'Edit Entry';
 
-}
+  // Issue#4: show the delete button
+  const $deleteButton = document.querySelector('button.delete');
+  $deleteButton.classList.remove('hidden');
 
+}
 $ulEntryList.addEventListener('click', editEntriesHandler);
+
+// Issue#4 the delete button functionality
+const $deleteButton = document.querySelector('button.delete');
+
+$deleteButton.addEventListener('click', deleteButtonHandler);
+
+function deleteButtonHandler(event) {
+  event.preventDefault();
```

Comment:
Not needed. We only really need `event.preventDefault` when working with `submit` events, because we want to stop the page from refreshing. In most other events, we don't actually care about preventing the default action